extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -719,4 +742,28 @@ export class Bind {
 
     return false;
   }
+
+
+  /**
+   * Wait for bind scan to finish for testing.
+   *
+   * @return {?Promise}
+   * @visibleForTesting
+   */
+  waitForInitializationForTesting() {
```

Comment:
Nit: `initializePromiseForTesting`

"waitForInitialization" is a verb and sounds like the method is actively doing something.